components/Main/Tuber/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import Tuber from '.'
+import type { Meta, Story } from '@storybook/react'
+import type { TuberProps } from '.'
+
+const meta: Meta = {
+  component: Tuber,
+  title: 'Tuber',
+}
+
+const Template: Story<TuberProps> = (args) => <Tuber {...args} />
+
+export const AGathering = Template.bind({})
+AGathering.args = {
```

Comment:
This part comes from Storybook's documents. It is recommended to use `args` [here](https://storybook.js.org/docs/react/writing-stories/introduction/) and [here](https://storybook.js.org/docs/react/writing-stories/introduction/). The advantage of using `args` is that makes Storybook handle controls automatically. 
I think it is quite OOP and tried to change it to an FP way like below, however, controls won't work: 
```js
// No controls
const src = 'https://www.youtube.com/embed/4jUMUcCuMJ4'
export const AGathering = (): JSX.Element => <Tuber src={src} />
```

Is there any other way to do this?

Reply:
Nah I think you need to use the args to get the controls to work. Personally I don't have too much of a problem with the args approach. Part of it is that this is part of Storybooks API so sort of out of our control really.

Reply:
Agreed.